app/logger/logs/complete.log

```diff
@@ -0,0 +1,9 @@
+11:17:06 - info: File: .keep does not match pattern: /\.js$/
```

Comment:
This log files shouldn\'t be in the repo, you could add this whole folder to the gitignore